user_tools/src/spark_rapids_pytools/resources/databricks_aws-configs.json

```diff
@@ -28,8 +28,7 @@
                   "value": "a65839fbf1869f81a1632e09f415e586922e4f80"
                 },
                 "size": 962685
-              },
```

Comment:
The `"type": "jar" `property did not conform to the `RuntimeDependency` type specification. Previously, we allowed extra keys to be included, which resulted in properties like type passing validation incorrectly.